src/chat-bubble/styles.scss

```diff
@@ -0,0 +1,59 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+@use "../../node_modules/@cloudscape-design/design-tokens/index.scss" as cs;
+@use "../internal/shared" as shared;
+
+.root {
+  @include shared.styles-reset;
+
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
```

Comment:
Need to use design tokens for this and other spacing, line height, and border radii values.

Reply:
Going to talk to designer about which ones should be tokens.